SimTKcommon/Geometry/src/DecorativeGeometryRep.h

```diff
@@ -568,12 +568,20 @@ class DecorativeMeshFileRep : public DecorativeGeometryRep {
 // no default constructor
 explicit DecorativeMeshFileRep(const std::string& meshFileName) 
 :   meshFile(meshFileName) {
+    pmesh = nullptr;
 }
 
 const std::string& getMeshFile() const {
     return  meshFile;
 }
 
+const PolygonalMesh& getMesh() {
```

Comment:
For the use case from OpenSim::Mesh, all the path searches, extension checks, and permission checks are performed before the DecorativeMeshFile is instantiated so the control path is relatively safe in that case (unless parsing the file fails or the file is invalid) but as an API that can be used by others in arbitrary contexts, I agree some validation and documentation would be needed. I'll add safety checks and doxygen docs.

I wasn't sure which one is cleaner (const and mutable or neither) I went with neither since mutable seems to turn red flags but I'll do what's consistent with Simbody conventions, so let me know your preference.